demos/topbars.html

```diff
@@ -19,709 +19,290 @@
-				<a class="topbar-link topbar-link-selected" href="#">Apps</a>
-			</li>
-			<li class="topbar-item">
-				<a class="topbar-link" href="#">Documentation</a>
-			</li>
-			<li class="topbar-item">
-				<a class="topbar-link" href="#">Docs</a>
-			</li>
-			<li class="topbar-item">
-				<a class="topbar-link" href="#">Faq</a>
-			</li>
-			<li class="topbar-item topbar-avatar">
-				<button class="topbar-link btn-transparent">
-					<object class="avatar-round avatar-small" data="http://www.gravatar.com/avatar/f159cc42cdb1ea7b0f757474cace8c15?d=blank&amp;s=120" type="image/jpeg">
-						<span class="avatar-initials avatar-small">Z</span>
-					</object>
-				</button>
-			</li>
-		</ul>
-	</nav>
-
-	<h1>Topbar Default: Logo Icon + Text</h1>
-
-	<nav class="topbar">
-		<div class="topbar-logo">
-			<a class="topbar-logo-link" href="#">
-				<span class="icon-12-github topbar-logo-icon"></span>
-				<h3 class="topbar-logo-text">Github</h3>
-			</a>
-		</div>
-		<ul class="topbar-list">
-			<li class="topbar-item">
-				<a class="topbar-link topbar-link-selected" href="#">Apps</a>
-			</li>
-			<li class="topbar-item">
-				<a class="topbar-link" href="#">Documentation</a>
-			</li>
-			<li class="topbar-item">
-				<a class="topbar-link" href="#">Docs</a>
-			</li>
-			<li class="topbar-item">
-				<a class="topbar-link" href="#">Faq</a>
-			</li>
-			<li class="topbar-item topbar-avatar">
-				<button class="topbar-link btn-transparent">
-					<object class="avatar-round avatar-small" data="http://www.gravatar.com/avatar/f159cc42cdb1ea7b0f757474cace8c15?d=blank&amp;s=120" type="image/jpeg">
-						<span class="avatar-initials avatar-small">Z</span>
-					</object>
-				</button>
-			</li>
-		</ul>
-	</nav>
-
-	<h1>Topbar Default: Logo Image</h1>
-
-	<nav class="topbar">
-		<div class="topbar-logo">
-			<a class="topbar-logo-link" href="#">
-				<img class="topbar-logo-image" src="logo.svg">
-			</a>
-		</div>
-		<div class="topbar-list-container">
-			<ul class="topbar-list">
-				<li class="topbar-item">
-					<a class="topbar-link topbar-link-selected" href="#">Apps</a>
-				</li>
-				<li class="topbar-item">
-					<a class="topbar-link" href="#">Documentation</a>
-				</li>
-				<li class="topbar-item">
-					<a class="topbar-link" href="#">Docs</a>
-				</li>
-				<li class="topbar-item">
-					<a class="topbar-link" href="#">Faq</a>
-				</li>
-				<li class="topbar-item topbar-avatar">
-					<button class="topbar-link btn-transparent">
-						<object class="avatar-round avatar-small" data="undefined" type="image/jpeg">
-							<span class="avatar-initials avatar-small">Z</span>
-						</object>
-					</button>
+	<h1>Topbar Dark: Logo Text</h1>
+
+	<header class="topbar topbar-dark">
+		<h1 class="topbar-logo">
+			<a class="topbar-logo-link" href="#"><span class="topbar-logo-text">Metal.js</span></a>
+		</h1>
+		<nav class="topbar-menu">
+			<button class="topbar-menu-toggle">Menu</button>
+			<ul class="topbar-list" id="nav-list">
```

Comment:
Is this ID `nav-list` needed?

Reply:
No, if the list will not be access. Yes, if it will be.

```html
<button aria-controls="SOME_ID">Menu</button>
<ul id="SOME_ID"/>
```

This is typically paired with `aria-expanded`. See:

https://codepen.io/2kool2/pen/kXVKKb (in depth)
https://codepen.io/stowball/pen/eBKvoO (simple)

Reply:
Ohhh perfect.